src/Core/Session.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace TrendingRepos\Core;
+
+class Session {
+    public function __construct()
+    {
+        if($this->isStarted())
```

Comment:
Shouldn't it be a negation here? 
```suggestion
        if(!$this->isStarted())
```

Reply:
Ops